bumble/gatt.py

```diff
@@ -202,6 +210,13 @@ def __init__(self, uuid, characteristics, primary=True):
         self.characteristics   = characteristics[:]
         self.primary           = primary
 
+    def get_service_advertising_data(self):
```

Comment:
`get_data` ?

Reply:
I am willing to make the change, but just feel `get_data` is a little bit ambiguous as there are many data in GATT, and this advertising_data is generated from service, not directly from GATT stored data. 
@barbibulle Do you have a better naming idea for this? :)

Reply:
I would recommend a compromise: call this `get_advertising_data` (omit the term `service` since this is a method on the `Service` class already, but make it clear that this returns the `Service Data` for advertising.

Reply:
ack